back-end/libs/common/src/utils/sdk/transaction.ts

```diff
@@ -200,10 +204,24 @@ export const computeSignatureKey = async (
             await mirrorNodeService.getAccountInfo(nodeAccountId, mirrorNetwork),
           );
           if (accountInfo?.key) {
-            signatureKey.push(accountInfo.key);
+            nodeSignatureKey.push(accountInfo.key);
+          }
+        }
+      }
+
+      if (transactionIs(NodeDeleteTransaction, transaction)) {
+        const COUNCIL_ACCOUNTS = ['0.0.2', '0.0.50', '0.0.55'];
```

Comment:
Is there a better place to store this const to be reused throughout the app? Instead of duplicating it.

Reply:
Just in case it were for some reason to change in the future (accounts added or removed)